src/data/complex/is_R_or_C.lean

```diff
@@ -49,21 +51,26 @@ class is_R_or_C (K : Type*) extends nondiscrete_normed_field K, normed_algebra 
 namespace is_R_or_C
 
 variables {K : Type*} [is_R_or_C K]
-local notation `𝓚` := @is_R_or_C.of_real K _
 local postfix `†`:100 := @is_R_or_C.conj K _
 
-lemma of_real_alg : ∀ x : ℝ, 𝓚 x = x • (1 : K) :=
-λ x, by rw [←mul_one (𝓚 x), smul_coe_mul_ax]
+-- see Note [coercion into rings]
+@[priority 900] instance cast_coe : has_coe_t ℝ K := ⟨of_real⟩
```

Comment:
Now that `of_real` is just `algebra_map`... Should we consider enabling this coercion for all algebras?

Reply:
Yes, I think that would make sense.

Reply:
https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/Notation.20for.20.60algebra_map.60.20.2F.20.60algebra.2Eof_id.60 is also relevent here

Reply:
So I guess the conclusion of that discussion is that we should keep this coercion local...